core/cmd/local_client.go

```diff
@@ -584,6 +589,11 @@ func (cli *Client) RebroadcastTransactions(c *clipkg.Context) (err error) {
 		return cli.errorOut(errors.Wrap(err, "error authenticating keystore"))
 	}
 
+	err = keyStore.Eth().CheckEnabled(address, chainID)
+	if err != nil && !skipAddressChecks {
```

Comment:
Does the check still need to run even if we are skipping and ignoring the erro?

Reply:
Resolved in [`b20efa1` (#8881)](https://github.com/smartcontractkit/chainlink/pull/8881/commits/b20efa1e8dea48c590486a9568ca844b16f41417)